api/tests/acceptance/application/airtable-proxy/airtable-proxy-controller-skill_test.js

```diff
@@ -159,7 +159,7 @@ describe('Acceptance | Controller | airtable-proxy-controller | create skill tra
   });
 });
 
-describe('Acceptance | Controller | airtable-proxy-controller | retrieve skill translations', () => {
+describe.skip('Acceptance | Controller | airtable-proxy-controller | retrieve skill translations', () => {
```

Comment:
```suggestion
describe('Acceptance | Controller | airtable-proxy-controller | retrieve skill translations', () => {
```

Même question que plus haut.